integration/tests/economy/test_economics.py

```diff
@@ -1321,23 +1482,135 @@ def test_scheduled_trx_for_erc20_for_spl(
         tree_acc_data.add_trx(trxs[2], 0xFFFF, 1)
         tree_acc_data.add_trx(trxs[3], 0xFFFF, 1)
 
-        sol_balance_before = operator.get_solana_balance()
-        token_balance_before = operator.get_token_balance(web3_client_sol)
-
-        evm_loader.create_tree_account_multiple(neon_user, treasury_pool, tree_acc_data.data)
+        evm_loader.create_tree_account_multiple(neon_user_with_all_tokens, treasury_pool, tree_acc_data.data)
         web3_client_sol.send_all_scheduled_transactions(trxs)
 
         for trx in trxs:
             check_trx_is_success(web3_client_sol, evm_loader, trx.hash().hex(), timeout=180)
-        sol_balance_after = operator.get_solana_balance()
-        token_balance_after = operator.get_token_balance(web3_client_sol)
+
+        operator_inner_balance_after = operator.get_token_balance(web3_client_sol)
+        operator_sol_balance_after = operator.get_solana_balance()
+
+        user_inner_sol_balance_after = web3_client_sol.get_balance(neon_user_with_all_tokens.checksum_address)
+        full_volume_after = operator_inner_balance_after + user_inner_sol_balance_after
+        diff_volume = full_volume_before - full_volume_after
+        assert diff_volume == 0, f"not same, diff={full_volume_before-full_volume_after}"
+
         token_price = web3_client_sol.get_token_usd_gas_price()
-        sol_diff = sol_balance_before - sol_balance_after
-        token_diff = web3_client_sol.to_main_currency(token_balance_after - token_balance_before)
+        sol_diff = operator_sol_balance_before - operator_sol_balance_after
+        token_diff = web3_client_sol.to_main_currency(operator_inner_balance_after - operator_inner_balance_before)
         assert_profit(sol_diff, sol_price, token_diff, token_price, web3_client_sol.native_token_name)
 
+    def test_scheduled_trx_send_value(
+        self,
+        web3_client_sol,
+        neon_user,
+        evm_loader,
+        treasury_pool,
+        operator,
+        sol_price,
+        web3_client,
+        operator_keypair,
+        common_contract,
+        neon_api_client,
+        treasury_pool_new,
+        second_operator_keypair,
```

Comment:
looks like you moved the test from neon_evm test
in operator tests we should sent trxs by proxy